Orange/widgets/classify/ownaivebayes.py

```diff
@@ -24,30 +24,15 @@ class OWNaiveBayes(OWProvidesLearner, widget.OWWidget):
 
     def __init__(self):
         super().__init__()
+        self.data = None
+        self.preprocessors = None
 
-        # GUI
         gui.lineEdit(gui.widgetBox(self.controlArea, self.tr("Name")),
                      self, "learner_name")
-
         gui.rubber(self.controlArea)
-
-        gui.button(self.controlArea, self, self.tr("&Apply"),
+        gui.button(self.buttonsArea, self, self.tr("&Apply"),
                    callback=self.apply)
-        self.controlArea.layout().addWidget(self.report_button)
-
-        self.data = None
-        self.preprocessors = None
-
-        self.initialize()
-
-    def initialize(self):
```

Comment:
I removed this because it duplicates `apply` and, besides, it's wrong since it outputs some kind of "default learner" without proper arguments.